examples/numgen.py

```diff
@@ -136,20 +141,19 @@ def receivedSlot(self, channelName: str, content: Any):
             if isinstance(content, dict):
                 self.updateDB(content)
             else:
-                print("The message for the channel db should be a dictionary.")
+                logger.info("The message for the channel db should be a dictionary.")
         else:
-            print(f"The message was ignored because "
-                  f"the treatment for the channel {channelName} is not implemented.")
+            logger.info("The message was ignored because "
+                  "the treatment for the channel %s is not implemented.", channelName)
 
     @pyqtSlot()
     def setDB(self):
         """Sets the database to store the number."""
         self.dbName = self.generatorFrame.dbBox.currentText()
         self.viewerFrame.statusLabel.setText("database updated")
-        self.broadcast(
-            "log", 
-            f"Database to store is set as {self.dbName}." if self.dbName
-            else "Database to store is not selected."
+        logger.info(
+            "Database to store is set as %s." if self.dbName
+            else "Database to store is not selected.", self.dbName
```

Comment:
It seems to make a syntax error.

```python
logger.info(
            "Database to store is set as %s.", self.dbName if self.dbName
            else "Database to store is not selected."
)
```

Isn't this correct?

Reply:
I found 
_logger.info(
            "Database to store is set as %s." % self.dbName if self.dbName
            else "Database to store is not selected."
)_
print message as we want.

But pylint tell me to use 
_logger.info(
          f"Database to store is set as {self.dbName}." if self.dbName
            else "Database to store is not selected."
)_ 
again. So I used it.

Reply:
Oh, then does Pylint say that my suggestion doesn't meet the Pylint condition?
I think not using the _f-formatting_ is better..

Reply:
As I said, please use `if-else` statement instead. I don't see any benefit from using the ternary expression.
In addition, `logger.info("%s" % x)` and `logger.info("%s", x)` are different! The former is just using the `%`-formatting, which should be replaced with the f-string as pylint complained. The latter is using the `logging` API. Comment questions if you need more explanation about this.

Reply:
In addition, previously the code used the ternary expression because it didn't use the `logging` API.

Reply:
@kangz12345 : I understand. I will use if-else instead.
@BECATRUE : When I print message as you suggested using `print` in python.idle, if `self.dbName == Sam`, it prints  _"Database to store is set as %s. Sam"_. But what we want is _"Database to store is set as Sam"_. I don't test for exact logger so I am not sure about case of logging.